hoot-services/src/test/java/hoot/services/controllers/job/ReviewResourcePrepareTest.java

```diff
@@ -187,7 +188,7 @@ public void testPrepareInvalidMapIdParam() throws Exception
   {
     ReviewTestUtils.createDataToPrepare();
 
-    final long fakeMapId = mapId + 2;
+    final long fakeMapId = (long)RandomNumberGenerator.nextDouble(mapId + 10^4, Integer.MAX_VALUE);
```

Comment:
@mingjiesu Where did this change come from? Did it sneak in from somewhere else?